ajax_with_jquery_exercise/style.css

```diff
@@ -0,0 +1,81 @@
+body {
+    font-family: "Verdana", sans-serif;
+    font-size: 14px;
+}
+
+/* navigation bar */
+ul {
+    list-style-type: none;
+}
+
+nav ul li {
+    display: inline;
+}
+
+nav {
+    background-color: #e3481c;
+    padding: 5px;
+}
+
+.logo {
+    font-weight: 600;
+    font-size: 18px;
+    padding-right: 2em;
+}
+
+.login:hover , .favs:hover {
+    text-decoration: underline;
+}
+
+/* aside - submission form */
+aside {
+    display: none;
+    background-color: #f5f5ee;
+    padding: 10px;
+    padding-left: 50px;
```

Comment:
If you are going to overwrite part of the padding above, just do it all on one line: `padding: TOP RIGHT BOTTOM LEFT`